From 7357766cd7003bae36c4cac37b74939662d09cdd Mon Sep 17 00:00:00 2001
From: Alexander Volkov <avolkov@astralinux.ru>
Date: Mon, 22 Jun 2020 13:25:06 +0300
Subject: [PATCH 3/5] simplematerial example: Fix double installation of
 main.qml

qt_example_installs.prf already generates install rules for all example
sources, including content of .qrc files.

This commit amends bbb6071dc69a2513d2586faaf9896841bf724786.

Task-number: QTBUG-84301
Change-Id: Idfef30501dfaaaddebf773aaa690206eedb458fc
Reviewed-by: Kai Koehne <kai.koehne@qt.io>

Conflicts:
	examples/quick/scenegraph/simplematerial/simplematerial.pro

(cherry picked from commit 7148f2117226f3e6ca63b5d7a1e8d8fec8475a0b)
Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 examples/quick/scenegraph/simplematerial/simplematerial.pro | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/examples/quick/scenegraph/simplematerial/simplematerial.pro b/examples/quick/scenegraph/simplematerial/simplematerial.pro
index 6ae935f357..12b1c43226 100644
--- a/examples/quick/scenegraph/simplematerial/simplematerial.pro
+++ b/examples/quick/scenegraph/simplematerial/simplematerial.pro
@@ -6,7 +6,4 @@ SOURCES += \
 RESOURCES += simplematerial.qrc
 
 target.path = $$[QT_INSTALL_EXAMPLES]/quick/scenegraph/simplematerial
-qml.files = main.qml
-qml.path = $$[QT_INSTALL_EXAMPLES]/quick/scenegraph/simplematerial
-
-INSTALLS += target qml
+INSTALLS += target
-- 
2.20.1

